### PR TITLE
refactor: check i18n key

### PR DESCRIPTION
**Pull request by BaojunCZ**

### CODE REVIEW

File: packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const InputWalletPasswordDialog = ({
     >
       {wallet ? (
         <>
-          <Card.Header>{t('settings.walletmanger.deletewallettitle', { name: wallet.name })}</Card.Header>
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Settings/InputWalletPasswordDialog.tsx]: wallet-manager

File: packages/neuron-ui/src/components/Transfer/index.tsx [hunk 2]
```diff
@@ -127,7 +127,7 @@ const Transfer = (props: React.PropsWithoutRef<ContentProps & RouteComponentProp
   return (
     <Container>
       <Card>
-        <Card.Header>{t('siderbar.send')}</Card.Header>
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/Transfer/index.tsx]: sidebar is a word

File: packages/neuron-ui/src/components/WalletDetail/index.tsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ const WalletDetail: React.SFC<{ children?: React.ReactNode } & Partial<ContentPr
   const { wallet } = useNeuronWallet()
   const [t] = useTranslation()
 
-  const items = [`${t('settings.walletmanger.walletdetail.balance')}: ${wallet.balance}`]
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/WalletDetail/index.tsx]: wallet-manager

File: packages/neuron-ui/src/components/WalletDetail/index.tsx [hunk 4]
```diff
@@ -22,7 +22,7 @@ const WalletDetail: React.SFC<{ children?: React.ReactNode } & Partial<ContentPr
       </ListGroup>
     </Card>
   ) : (
-    <Alert variant="warning">{t('settings.walletmanger.walletdetail.nowallet')}</Alert>
```
Comment (Keith-CY) [inline packages/neuron-ui/src/components/WalletDetail/index.tsx]: wallet-manager

Comment (Keith-CY) [reply packages/neuron-ui/src/components/WalletDetail/index.tsx]: better to be in the group of messages

**Comment by BaojunCZ** [comment]
fixed

**Pull request merged**